docs/Install-with-Docker.md

```diff
@@ -33,11 +33,19 @@ Using Docker will have the same benefits as using virtual machines, but with muc
 
 ![Docker containers vs Virtual machines](https://user-images.githubusercontent.com/75517/51078179-d4fec680-16b1-11e9-8ce6-87b5053f55dd.png)
 
-## Setup
+## PART 1 - Setup
 
 ### Software requirements
 
-The only requirements are [git](https://git-scm.com/download/) and Docker.
+The only requirements are [git](https://git-scm.com/download/) and Docker (https://docs.docker.com/docker-for-windows/install/).
```

Comment:
why inserting this link here? the link is specific for Windows

Reply:
The old guide before had no links, so the guide having a windows link is an improvement, if someone else who uses MAC or Linux wants to add a linux link they can but I don't have them.

Reply:
@JamesGent yes there are links O_o

![image](https://user-images.githubusercontent.com/75517/98540046-97f3c300-228d-11eb-8d52-1891934e5282.png)